plugin/api.go

```diff
@@ -86,6 +86,11 @@ type API interface {
 	// GetTeamByName gets a team by its name.
 	GetTeamByName(name string) (*model.Team, *model.AppError)
 
+	// GetTeamsUnreadForUser will return an array with TeamUnread objects that contain the amount
+	// of unread messages and mentions the current user has for the teams it belongs to.
+	// An optional team ID can be set to exclude that team from the results. Must be authenticated.
+	GetTeamsUnreadForUser(userId, teamIdToExclude string) ([]*model.TeamUnread, *model.AppError)
```

Comment:
I think "Must be authenticated." should be removed.

Reply:
You're so right 🙈

Reply:
Might suggest simplifying to:
> GetTeamsUnreadForUser gets the unread message and mention counts for each team to which the given user belongs.

Generally talking about `arrays` and the model objects in question is superfluous given the method signature, and it's more useful IMO to focus on the semantics.